launcher/ui/dialogs/ResourceUpdateDialog.cpp

```diff
@@ -264,6 +272,7 @@ void ResourceUpdateDialog::checkCandidates()
 auto ResourceUpdateDialog::ensureMetadata() -> bool
 {
     auto index_dir = indexDir();
+    auto index_dir_resolver = std::bind(&resolveIndexDir, m_resourceModel.get(), std::placeholders::_1);
```

Comment:
I meant soomething like this:
```suggestion
    auto indexDirResolver =
    std::bind(&ResourceFolderModel::indexDirForResource,
              m_resourceModel.get(),
              std::placeholders::_1);
```
But if you think it is worse please go back to just using lambda instead of an external fucntion

Reply:
also camelCase is prefered